pkgs/by-name/ca/calibre/package.nix

```diff
@@ -222,6 +222,7 @@ stdenv.mkDerivation (finalAttrs: {
       $ETN 'test_qt'  # we don't include svg or webp support
       $ETN 'test_import_of_all_python_modules'  # explores actual file paths, gets confused
       $ETN 'test_websocket_basic'  # flakey
+      $ETN 'test_recipe_browser_webengine' # hangs on cuda
```

Comment:
```suggestion
      # hangs with cuda enabled, also:
      # eglInitialize: Failed to get system egl display
      # Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
      $ETN 'test_recipe_browser_webengine' 
      
```

Reply:
documenting these issues may enable the next contributor to solve the fundamental issue ;)

likely it is the lack of a dbus and a means to open a gl context in the sandbox that is the issue. xvbf-run may be part of the solution, but this requires experimentation.